Makefile

```diff
@@ -31,13 +34,16 @@ ifneq ("$(wildcard $(SCSS_DIR320)/gtk-dark.scss)","")
 else
 	cp "$(DIST_DIR320)/gtk.css" "$(DIST_DIR320)/gtk-dark.css"
 endif
-css: css_gtk3 css_gtk320
+css_cinnamon:
+	mkdir -p $(DIST_DIR_CINNAMON)
+	$(SASS) $(SASSFLAGS) "$(SCSS_DIR_CINNAMON)" "$(SCSS_DIR_CINNAMON)/cinnamon.scss" "$(DIST_DIR_CINNAMON)/cinnamon.css"
+css: css_gtk3 css_gtk320 css_cinnamon
```

Comment:
instead add `css_connamon` to `all:` target

Reply:
i'll also change later both change_color.sh and oomox gui to add to `HiDPI` to 2 new export options:
- [ ] Export only the current GTK version
- [ ] Export Cinnamon theme

those settings are saved between oomox gui restarts

Reply:
Done

Reply:
thanks!

Reply:
i'll make change for oomox gui tonight (without that change cinnamon theme can be exported only directly with change_color.sh)